libs/entry-components/styles/modules/components/tables/_rows.scss

```diff
@@ -2,6 +2,8 @@
 @use '@enigmatry/scss-foundation/src/modules/lists/row-coloring' as list;
 
 @mixin generate-from($theme) {
+	$selected-row-background: map.get($theme, 'tables', 'rows', 'odd-even-background-selected');
```

Comment:
Keep variables close to their usage, in other words, please move this between line 15 and 16.